web/pipeline/views/general.py

```diff
@@ -37,24 +38,13 @@ class CommunityList(generics.ListAPIView):
 
 
 class ServiceList(generics.ListAPIView):
+    serializer_class = ServiceListSerializer
+    pagination_class = LimitOffsetPagination
 
-    queryset = Service.objects.all()
-
-    def get(self, request, format=None):
```

Comment:
overriding `get()`, which is more low level, removes some features that would be automatically handled by DRF, such as pagination